editors/code/package.json

```diff
@@ -949,6 +954,11 @@
                         "type": "string"
                     }
                 },
+                "rust-analyzer.enableBackspaceListener": {
+                    "markdownDescription": "Enable the automatically remove whilespace functionality when press backspace key",
+                    "type": "boolean",
+                    "default": true
```

Comment:
We should probably disable this by default. It will most likely be confusing to people to have this behavior in their opened documents in one language but not another.

Reply:
This was supposed to be a feature implemented by the editor, unfortunately not supported by vscode. I think it should be enable by default (I can turn it off if you insist)